tests/resources/SeleniumUtil.robot

```diff
@@ -47,3 +47,31 @@ Init Chrome Driver
     Run Keyword If  '${out[0]}'=='FAIL'  Capture Page Screenshot
     Should Be Equal As Strings  '${out[0]}'  'PASS'
     Sleep  5
+
+Init Chrome Driver And Set Up Download Directory
```

Comment:
Can we modify original keyword than create a new one?

Reply:
I can change the old keyword, but I think it's too big a change, because there are a lot of TestCases that reference the old keyword, I can change the old keyword if you think it's ok to change the old keyword